packages/interface/src/components/complementary-area/index.js

```diff
@@ -295,47 +295,51 @@ function ComplementaryArea( {
 				scope={ scope }
 				id={ identifier.replace( '/', ':' ) }
 			>
-				<ComplementaryAreaHeader
-					className={ headerClassName }
-					closeLabel={ closeLabel }
-					onClose={ () => disableComplementaryArea( scope ) }
-					smallScreenTitle={ smallScreenTitle }
-					toggleButtonProps={ {
-						label: closeLabel,
-						size: 'small',
-						shortcut: toggleShortcut,
-						scope,
-						identifier,
-					} }
-				>
-					{ header || (
-						<>
-							<h2 className="interface-complementary-area-header__title">
-								{ title }
-							</h2>
-							{ isPinnable && (
-								<Button
-									className="interface-complementary-area__pin-unpin-item"
-									icon={ isPinned ? starFilled : starEmpty }
-									label={
-										isPinned
-											? __( 'Unpin from toolbar' )
-											: __( 'Pin to toolbar' )
-									}
-									onClick={ () =>
-										( isPinned ? unpinItem : pinItem )(
-											scope,
-											identifier
-										)
-									}
-									isPressed={ isPinned }
-									aria-expanded={ isPinned }
-									size="compact"
-								/>
-							) }
-						</>
-					) }
-				</ComplementaryAreaHeader>
+				{ header !== false && (
```

Comment:
This is the only change here - if header is explicitly set to false then we don't output it. This might break other things.

Reply:
I don't see a better way to do this either.